pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -148,6 +167,8 @@ type smcpmutator interface {
 	GetProfiles() []string
 	SetProfiles(profiles []string)
 	GetPatches() []jsonpatch.JsonPatchOperation
+	IsOpenShiftRouteEnabled() *bool
```

Comment:
nit: I still think returning a `bool` makes more sense here. Either it is set, or it isn't, a third option isn't required

Reply:
When it returns `nil`, it is the user doesn't specify any value for the field and it is also when we want to set the default. I want to know if it is the third option.